test/telemetry/src/main/java/org/opensearch/test/telemetry/tracing/StrictCheckSpanProcessor.java

```diff
@@ -60,7 +60,8 @@ private MockSpanData toMockSpanData(Span span) {
             System.nanoTime(),
             false,
             span.getSpanName(),
-            Thread.currentThread().getStackTrace()
+            Thread.currentThread().getStackTrace(),
+            ((MockSpan) span).getAttributes()
```

Comment:
Let's be safe here please:

```suggestion
         (span instanceof  MockSpan)  ?  ((MockSpan) span).getAttributes() : Map.of()
```